.github/workflows/build.yml

```diff
@@ -209,12 +209,13 @@ jobs:
     strategy:
       matrix:
         build_type: [Debug, Release]
+        machine: ["x64"]
```

Comment:
Is it possible to just use `machine: ["x64", "arm64"]` instead of duplicating the windows configuration?

Reply:
Yes, we should write it without duplication. Thanks for noticing!